lib/resources/shim-head.js

```diff
@@ -2,6 +2,25 @@
 ((win) => {
   win.ELECTRON = true;
 
+  if (process.env.ELECTRON_PROTOCOL_SERVE_INDEX && window.location.protocol !== 'file:') {
```

Comment:
I should not have made it possible to customize that protocol. However, in its current iteration the function return the name of the protocol, if that is any help. so calling `protocolServe()` returns `serve`.